playbooks/init/sanity_checks.yml

```diff
@@ -32,3 +32,11 @@
   - fail: msg="Currently, NetworkManager must be installed and enabled prior to installation."
     when:
     - not ('ActiveState=active' in nm_show.stdout) | bool
+
+  - name: Verify SELinux is enabled
+    fail:
+      msg: "SELinux is disabled, This deployment type requires that SELinux is enabled."
+    when:
+    - (ansible_selinux is not defined or ansible_selinux.status != 'enabled')
```

Comment:
https://docs.ansible.com/ansible/latest/modules/selinux_module.html#return-values

Valid values are `enforcing` and `permissive`

Reply:
However it appears to have worked previously :/

Reply:
This is using the ansible vars, not the module:
```json
        "ansible_selinux": {
            "config_mode": "enforcing",
            "mode": "enforcing",
            "policyvers": 31,
            "status": "enabled",
            "type": "targeted"
        },
```
"status" is `enabled`

Reply:
Looks like maybe we should check for more than just status enabled.
```json
        "ansible_selinux": {
            "config_mode": "enforcing",
            "mode": "permissive",
            "policyvers": 31,
            "status": "enabled",
            "type": "targeted"
        },
```